src/xcode/ENA/ENA/Source/Scenes/TestresultAvailable/TestresultAvailableViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+struct TestresultAvailableViewModel {
```

Comment:
Throughout the app, I have seen that we use camel case for `TestResult`, so maybe rename the struct to `TestResultAvailableViewModel`, what do you think?